internal/provider/provider.go

```diff
@@ -79,16 +79,22 @@ func (p *BloxOneProvider) Configure(ctx context.Context, req provider.ConfigureR
 func (p *BloxOneProvider) Resources(ctx context.Context) []func() resource.Resource {
 	return []func() resource.Resource{
 		ipam.NewIpSpaceResource,
+
```

Comment:
Do we need this space? Same with other spaces here and in Datasource.

Reply:
Not needed, but thought it would be better for separating between packages

Reply:
Ok, then lets keep it as is